backend/prisma/migrations/20240729210931_init_favourites_table/migration.sql

```diff
@@ -0,0 +1,21 @@
+-- CreateTable
+CREATE TABLE "Favourite" (
+    "id" SERIAL NOT NULL,
+    "UserId" INTEGER NOT NULL,
```

Comment:
UserId is the id(number) of the user currently using the account and making the favorite selection